tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0032_partition_tables_by_year_and_month.py

```diff
@@ -1,9 +1,5 @@
 """
-Partition the event_sink.user_profile table
-
-.. pii: Stores Open edX user profile data.
-.. pii_types: user_id, name, username, location, phone_number, email_address, birth_date, biography, gender
-.. pii_retirement: local_api, consumer_api
+Partition the xapi table by year and month
```

Comment:
Everything in this file is just cleaning up incorrect language from the original commit.